### DISCUSSION TITLE
MyCode range in justMyCode

### DISCUSSION THREAD

**Discussion by sunyonghua** [microsoft/debugpy#1632] [Q&A]
MyCode range in justMyCode
Can I customize the range of justMyCode?

**Comment by sunyonghua** [comment]
![image](https://github.com/user-attachments/assets/49553fef-8cc8-405f-9a93-0854c97eada4)
Is just MyCode not debugging the official library?

**Comment by rchiodo** [comment]
Sorry I'm not sure what `MyCode` means. JustMyCode means anything in your current workspace in VS code can be debugged. If you set it to false, then you should be able to set breakpoints and step through any python package.

**Reply by sunyonghua** [reply]
I want to know if it is possible to block some folders in the VSCode workspace when the justMyCode is true

**Reply by rchiodo** [reply]
You mean mark some folders as not being your own code? Sorry that's not currently possible.

**Reply by rchiodo** [reply]
As a workaround (assuming your code was structured this way), you could split your code up by packages. Install just some of the packages into the venv. Then debug the code that wasn't installed. That would effectively mark the installed packages as 3rd party or not your code.

**Reply by sunyonghua** [reply]
Okay, I understand. Thank you
